src/components/InputField.jsx

```diff
@@ -0,0 +1,30 @@
+import { Flex, Input } from "@chakra-ui/react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+
+export function InputField({ placeholder }) {
+  return (
+    <Flex
+      className="search"
+      w="40vw"
```

Comment:
Why is the width set to this specific value `40vw`? Typically, we don't pass width for an element as a view port unit.